ironfish/src/wallet/wallet.ts

```diff
@@ -1606,13 +1606,15 @@ export class Wallet {
     account: Account,
     options?: {
       resetCreatedAt?: boolean
+      resetScanningEnabled?: boolean
       tx?: IDatabaseTransaction
     },
   ): Promise<void> {
     const newAccount = new Account({
       accountValue: {
         ...account,
         createdAt: options?.resetCreatedAt ? null : account.createdAt,
+        scanningEnabled: options?.resetScanningEnabled ? true : account.scanningEnabled,
```

Comment:
Should this change the flag? It's probably fine just pointing out this implication.